Replay/Replay/ViewController.swift

```diff
@@ -140,25 +152,27 @@ class ViewController: UIViewController {
     // MARK:
     // MARK: Playback
     
-    func startPlayer(forAsset asset: AVURLAsset?) -> Void {
-        
-        guard let mediaAsset = asset else {
+    func startPlayer(forAsset asset: [AVURLAsset]) -> Void {
+        var playerItemArray = [AVPlayerItem]()
+        if(asset.isEmpty){
```

Comment:
Lets make use of guard here. That way we can initialize playerItemArray only when asset array is not empty.